packages/orchestration/src/examples/stakeAtom.contract.js

```diff
@@ -50,25 +73,40 @@ export const start = async (zcf, privateArgs, baggage) => {
     zcf,
   );
 
+  /** @type {BrandToIssuer} */
+  const brandToIssuer = zone.mapStore('brandToIssuer');
+  for (const [keyword, brand] of Object.entries(brands)) {
+    brandToIssuer.init(brand, issuers[keyword]);
+  }
+
   async function makeAccount() {
     const account = await E(orchestration).makeAccount(
       hostConnectionId,
       controllerConnectionId,
     );
-    // #9212 TODO do not fail if host does not have `async-icq` module;
-    // communicate to OrchestrationAccount that it can't send queries
-    const icqConnection = await E(orchestration).provideICQConnection(
-      controllerConnectionId,
-    );
-    const accountAddress = await E(account).getAddress();
-    trace('account address', accountAddress);
-    const { holder, invitationMakers } = makeStakingAccountKit(
+
+    // TODO #9063, #9212 this should come from Chain object
+    const icqConnection = icqEnabled
+      ? await E(orchestration).provideICQConnection(controllerConnectionId)
+      : undefined;
```

Comment:
Maybe it's ok by the new [await safety style](https://github.com/Agoric/agoric-sdk/wiki/No-Nested-Await), but I've picked up an aversion to code that sometimes represents a turn boundary and sometimes doesn't.

```suggestion
    const icqConnection = await (icqEnabled
      ? E(orchestration).provideICQConnection(controllerConnectionId)
      : undefined);
```

Reply:
I'm also reminded to check for a clear commit point, as in the [prepare / commit pattern](https://github.com/Agoric/agoric-sdk/wiki/Prepare-Commit).

If `makeAccount()` succeeds but `provideICQConnection()` fails, should we release the account?